audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -929,12 +936,14 @@
       },
       "outputs": [],
       "source": [
-        "def requests(config: cloud_speech.RecognitionConfig, s: list) -> list:\n",
-        "    yield config\n",
+        "def requests(request_config: cloud_speech.RecognitionConfig, s: list) -> list:\n",
```

Comment:
Should the type be `RequestConfig` or `RecognitionConfig`?

Reply:
RecognitionConfig